### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by Lul-Abdifan**
### In this milestone,I did the following:

-   Create a table named vets
-   Create a "join table" called specializations for many-many relationship of species and vets.
-   Create a "join table" called visits for many-many relationship of animals and vets
-   Insert the following data for vets [HERE](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md)
-   Insert the following data for specializations [HERE](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md)
-   Insert the following data for visits [HERE](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md)
-  Write the queries for the given [HERE](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md)

Thank you for your understanding and patience. If you have any questions or concerns, please don't hesitate to reach out to me at @Lul-Abdifan.

### The screenshots are as follows:

q1
q2
q3
Q4
q5
q6
q7
q8
q9

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -129,5 +129,104 @@ JOIN owners ON animals.owner_id = owners.id
...
+
+
+
+
+
+
+
+
+
+
```
Comment (alexander16108) [inline queries.sql:219]: Hello there 👋🏾 the result of the following queries are incorrect please check your database and try again 👍🏾  thank you

- Who was Maisy Smith's first visit?
- How many times visits were with a vet that did not specialize in that animal's species?

the result of the first query ought to be `boarmon`
and the development of the second ought to be 12 unspecialized visits

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Lul-Abdifan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Lul-Abdifan) [reply queries.sql:219]: @alexander16108 Okay, I will fix it . Thank you.

**Comment by Lul-Abdifan** [comment]
### Changes Done

In these changes

- Fix error with Who was Maisy Smith's first visit? "
q12

- I have fixed  "How many times visits were with a vet that did not specialize in that animal's species?"
q11

**Comment by arslanbisharat** [comment]
Hi @Lul-Abdifan,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lul-Abdifan** [comment]
@arslanbisharat  Thank you for your approval .

**Pull request merged**